### PR TITLE
The syndicate depo closests actually use their sprites + door sprites.

### PR DESCRIPTION
**Pull request by 1080pCat**
## What Does This PR Do
Adds door sprites for the syndicate depo closets, so we can actually use the sprites in the depo.

## Why It's Good For The Game
We've had the closet sprites for god knows how long, but no *doors*. So i added them!
This allowed me to change the sprites to be on theme. And so the description of the closet doesn't lie about its colour.

## Images of changes

dreamseeker_Sds61jxfXd

image

## Testing
see above

<hr>

### Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/docs/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  
  
  <hr>

## Changelog

:cl:
tweak: The closets in the syndicate depo now use their respective sprites, that have existed for god knows how long.
imageadd: Added door sprites for the syndicate depo closets.
/:cl:

### CODE REVIEW

**Comment by Christasmurf** [comment]
https://github.com/ParadiseSS13/Paradise/pull/26832 
This PR makes changes to lockers, I would suggest waiting until it is merged and then updating the sprites used here if needed

**Comment by 1080pCat** [comment]
> #26832 This PR makes changes to lockers, I would suggest waiting until it is merged and then updating the sprites used here if needed

Aye, fair enough, drafting until the pr is merged

**Review by Henri215 — APPROVED** [review]

**Review by DGamerL — APPROVED** [review]

**Pull request merged**
